### ISSUE TITLE
could not find `serde` in the list of imported crates

### ISSUE THREAD

**Issue by mischnic** [swc-project/swc#7148]
```toml
[dependencies]
swc_ecmascript = { version = "0.223.0", features = ["parser", "transforms", "module", "optimization", "react", "typescript", "utils", "visit", "codegen", "utils", "preset_env"] }
swc_common = { version = "0.29.40", features = ["tty-emitter", "sourcemap"] }
swc_atoms = "0.4.39"
```

```
error[E0433]: failed to resolve: could not find `serde` in the list of imported crates
  --> /Users/niklas/.cargo/registry/src/github.com-1ecc6299db9ec823/swc_ecma_ast-0.101.0/src/class.rs:19:1
   |
19 | #[ast_node]
   | ^^^^^^^^^^^ could not find `serde` in the list of imported crates
   |
   = note: this error originates in the attribute macro `ast_node` (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by kdy1** [comment]
Oh bump script did something wrong. `ast_node` should be published as a breaking change, so `swc_common` cannot be `x.x.non-0`...

**Comment by kdy1** [comment]
Oops I did `ast_macro --breaking` at https://github.com/swc-project/swc/pull/7138#pullrequestreview-1355917160

https://github.com/swc-project/swc/pull/7109 will fix this

**Comment by kdy1** [comment]
This should be fixed by `swc_core = "0.71.0"`
